comp_sci_gender_bias/utils/io.py

```diff
@@ -0,0 +1,29 @@
+import pathlib
+from typing import Any
+import pickle
+
+
+def make_path_if_not_exist(path: pathlib.Path):
+    """If the path does not exist, make it"""
+    if not path.exists():
+        path.mkdir(parents=True)
+
+
+def save_pickle(obj: Any, save_path: pathlib.Path, file_name: str):
+    """Save object as pickle file
+
+    Args:
+        obj: Object to save as pickle file
+        save_path: Path to save file to
+        file_name: Name to give file
+    """
+    make_path_if_not_exist(save_path)
+    with open(save_path / file_name, "wb") as out:
```

Comment:
I'm used to input paths being allowed to be `str`. Is this good practice then to only allow then to be `pathlib.Path`?

Reply:
I've updated these functions to accept strings and `pathlib.Path` 🙂